docs/src/test-global-setup-teardown-js.md

```diff
@@ -3,13 +3,32 @@ id: test-global-setup-teardown
 title: "Global setup and teardown"
 ---
 
-There are two ways to configure global setup and teardown: using a global setup file and setting it in the config under [`globalSetup`](#configure-globalsetup-and-globalteardown) or using [project dependencies](#project-dependencies). With project dependencies, you define a project that runs before all other projects. This is the recommended way to configure global setup as with Project dependencies your HTML report will show the global setup, trace viewer will record a trace of the setup and fixtures can be used.
+There are two ways to configure global setup and teardown: using a global setup file and setting it in the config under [`globalSetup`](#configure-globalsetup-and-globalteardown) or using [project dependencies](#project-dependencies). With project dependencies, you define a project that runs before all other projects and a teardown project that runs after the dependencies. This is the recommended way to configure global setup as with Project dependencies your HTML report will show the global setup, trace viewer will record a trace of the setup and fixtures can be used.
```

Comment:
I'd drop the `and a teardown project that runs after the dependencies` since it's not really clear.

From the `teardown` project point of view, it runs after the **dependant** projects of the associated `setup`. 

It's hard to formulate when the teardown runs from the point of view of some other dependent project.